scene/gui/popup_menu.h

```diff
@@ -203,8 +211,10 @@ class PopupMenu : public Popup {
 
 	void _notification(int p_what);
 	bool _set(const StringName &p_name, const Variant &p_value);
-	bool _get(const StringName &p_name, Variant &r_ret) const;
-	void _get_property_list(List<PropertyInfo> *p_list) const;
+	bool _get(const StringName &p_name, Variant &r_ret) const { return property_helper.property_get_value(p_name, r_ret); }
+	void _get_property_list(List<PropertyInfo> *p_list) const { property_helper.get_property_list(p_list, items.size()); }
+	bool _property_can_revert(const StringName &p_name) const { return property_helper.property_can_revert(p_name); }
+	bool _property_get_revert(const StringName &p_name, Variant &r_property) const { return property_helper.property_get_revert(p_name, r_property); }
```

Comment:
I just realized this will be the same for pretty much every class, so it can be replaced with a macro 🤔
Though that's probably too extreme.

Reply:
Yeah I thought about this too while reviewing the code, but I also thought it might be a bit overkill.

I guess you'll see as you keep implementing it, maybe with 5-6 repetitions is starts making sense to make it a macro. The main issue is that it hides the fact that the class overrides some common Object virtual methods.